samples/wifi/twt/src/main.c

```diff
@@ -64,7 +65,12 @@ static bool wait_for_twt_resp_received(void)
 	for (i = 0; i < timeout_polls; i++) {
 		k_sleep(K_MSEC(STATUS_POLLING_MS));
 		if (twt_resp_received) {
-			return true;
+			/* check for twt resp as accept, if not return failure */
```

Comment:
What is AP says `suggest` shouldn't we demonstrate how to handle that case to our customers?